templates/pages/setup/general/dbreplica_setup.html.twig

```diff
@@ -52,10 +58,113 @@
-   {{ fields.htmlField('', replica_delays, '', {
-      no_label: true,
-      full_width: true
-   }) }}
 
    {{ inputs.hidden('_dbslave_status', 1) }}
+</div>
+
+<div class="hr-text">
+    <i class="ti ti-check"></i>
+    <span>{{ __('Replication status') }}</span>
+</div>
+
+{% set primary_ok  = replication_status['primary']['file'] %}
+{% set primary_unknow = replication_status['primary']['error'] is defined and replication_status['primary']['error'] is not null %}
+
+<div class="row ps-2 mb-3">
+   <div class="col-5">
+      <div class="card">
+         <div class="card-body">
+            <div class="accordion" id="accordion-primary">
+               <div class="accordion-item">
+                  <h2 class="accordion-header" id="heading-primary">
+                     <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapse-primary" aria-expanded="false" aria-controls="collapse-primary">
+                        <div class="col-auto">
+                           <span class="status-indicator status-{{ primary_ok ? 'green' : primary_unknow ? 'muted' : 'red' }} status-indicator-animated">
+                           <span class="status-indicator-circle"></span>
+                           <span class="status-indicator-circle"></span>
+                           <span class="status-indicator-circle"></span>
+                           </span>
+                        </div>
+                        <div class="col">
+                           <h3><span class="badge bg-{{ primary_ok ? 'green' : primary_unknow ? 'muted' : 'red' }}-lt">{{ primary_ok ? 'Up' : primary_unknow ? 'Unkown' : 'Down' }}</span><span class="mx-2">Primary ({{ primary_dbhost }})</span>{{ primary_unknow ? '' : replication_status['primary']['read_only'] == 'OFF' ? '<span class="badge badge-outline text-green">Writable</span>' : '<span class="badge badge-outline text-blue">Read-Only</span>' }}</h3>
+                        </div>
+                     </button>
+                  </h2>
+                  <div id="collapse-primary" class="accordion-collapse collapse" aria-labelledby="heading-primary" data-bs-parent="#accordion-primary">
+                     <div class="accordion-body">
+                        {% if primary_unknow %}
+                           <div class="alert alert-danger" role="alert">
+                              <span class="text-secondary">{{ __('MySQL server returned an error') }}:</span>
+                              <p class="mt-2">{{ replication_status['primary']['error'] }}</p>
+                           </div>
+                        {% else %}
+                           <ul class="p-0 list-unstyled space-y-3">
+                              <li><span class="text-secondary">Server ID:</span> {{ replication_status['primary']['server_id'] }}</li>
```

Comment:
Is that intended those strings are not translatable?

Reply:
I told myself that as these are technical terms, linked to MySQL variables which are in English, the translation does not make sense.